pkgs/tools/typesetting/sile/default.nix

```diff
@@ -39,8 +39,8 @@ stdenv.mkDerivation rec {
     ];
   };
 
-  # TODO: needs to tweak Makefile-fonts to avoid download fonts
-  doCheck = false; /*stdenv.targetPlatform == stdenv.hostPlatform
+  /*
+  stdenv.targetPlatform == stdenv.hostPlatform
```

Comment:
This change is surprising. The comment was a previous value of the `doCheck` attribute. Now it becomes very mysterious.

If you want to enable the check, you should state `doCheck = true;` and maybe fix the `checkTarget` value (and add `poppler` to the `checkInputs`).

Reply:
I assumed this was only being set at all because the default was true. I'll correct that.